src/pages/journal/journal.effects.ts

```diff
@@ -62,9 +73,9 @@ export class JournalEffects {
   journal$ = this.actions$.pipe(
     ofType(journalActions.LOAD_JOURNAL_SILENT),
     switchMap(
-      () => this.callJournalProvider$().pipe(
-        catchError(err => {
-          console.error(err);
+      () => this.callJournalProvider$(JournalRefreshModes.AUTOMATIC).pipe(
+        catchError((err) => {
+          this.analytics.logError('AutomaticJournalRefresh', err.message);
```

Comment:
Probably keep the console log here, we'll lose visibility if it only goes to analytics.